fluster/decoders/gstreamer.py

```diff
@@ -19,13 +19,25 @@
 # Boston, MA 02111-1307, USA.
 
 import shlex
+import subprocess
 from functools import lru_cache
 
 from fluster.codec import Codec, PixelFormat
 from fluster.decoder import Decoder, register_decoder
 from fluster.utils import file_checksum, run_command, normalize_binary_cmd
 
-PIPELINE_TPL = '{} filesrc location={} ! {} ! {} ! filesink location={}'
+PIPELINE_TPL = '{} filesrc location={} ! {} ! {} ! {} location={}'
+
+
+@lru_cache(maxsize=None)
+def gst_element_exists(element: str) -> bool:
+    '''Check if an element exists in current GStreamer installation'''
+    try:
+        subprocess.run(
+            ['gst-inspect-1.0', '--exists', element], stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
+        return True
+    except (subprocess.CalledProcessError, FileNotFoundError):
+        return False
```

Comment:
Maybe I'm too pedantic, but I feel safer if we insert a final statement with `return False` in case a different exception is ever thrown
```suggestion
        return False
    return False
```

Reply:
no no, you are not, I was catching all exception initially, and but the CI complain, and forgot that we could endup returning nothing.